README.md

```diff
@@ -1,61 +1,42 @@
 # TFS-Pandas
+
 [![Cron Testing](https://github.com/pylhc/tfs/workflows/Cron%20Testing/badge.svg)](https://github.com/pylhc/tfs/actions?query=workflow%3A%22Cron+Testing%22)
 [![Code Climate coverage](https://img.shields.io/codeclimate/coverage/pylhc/tfs.svg?style=popout)](https://codeclimate.com/github/pylhc/tfs)
 [![Code Climate maintainability (percentage)](https://img.shields.io/codeclimate/maintainability-percentage/pylhc/tfs.svg?style=popout)](https://codeclimate.com/github/pylhc/tfs)
 [![GitHub last commit](https://img.shields.io/github/last-commit/pylhc/tfs.svg?style=popout)](https://github.com/pylhc/tfs/)
 [![GitHub release](https://img.shields.io/github/release/pylhc/tfs.svg?style=popout)](https://github.com/pylhc/tfs/)
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.5070986.svg)](https://doi.org/10.5281/zenodo.5070986)
 
-This package provides reading and writing functionality for **table format system (tfs)** files. 
-
-## Getting Started
-
-### Prerequisites
+This package provides reading and writing functionality for [**table format system (tfs)** files](http://mad.web.cern.ch/mad/madx.old/Introduction/tfs.html). 
+Files are read into a `TfsDataFrame`, a class built on top of the famous `pandas.DataFrame`, which in addition to the normal behaviour attaches an `OrderedDict` of headers to the `DataFrame`.
 
-The package depends heavily on `pandas` and also on `numpy`, so these packages need
-to be installed in your python environment.
+## Installing
 
-### Installing
-
-Installation is easily done via `pip`. The package is then used as `tfs`.
+Installation is easily done via `pip`.
 
 ```
 pip install tfs-pandas
 ```
 
-Example:
+## Example Usage
 
+ The package is imported as `tfs`, and exports top-level functions for reading and writing:
 ```python
 import tfs
 
-data_frame = tfs.read('path_to_input.tfs', index="index_column")
-tfs.write('path_to_output.tfs', data_frame, save_index="index_column")
+data_frame = tfs.read("path_to_input.tfs", index="index_column")
+# do things with the data
+tfs.write("path_to_output.tfs", data_frame, save_index="index_column")
```

Comment:
maybe add here how to access headers since this is the distinguishing feature to regular `pd.DataFrame`